apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingVedtakRoute.kt

```diff
@@ -17,18 +18,60 @@ internal fun Route.behandlingVedtakRoute(
     oppgaveService: OppgaveServiceNy,
     behandlingService: BehandlingService
 ) {
-    route("/fattvedtak-behandling") {
+    route("/fattvedtak") {
         post {
-            val attesterVedtakOppgave = call.receive<AttesterVedtakOppgave>()
+            val fattVedtak = call.receive<VedtakEndringDTO>()
             val behandling = behandlingService.hentBehandling(
-                UUID.fromString(attesterVedtakOppgave.attesteringsOppgave.referanse)
+                UUID.fromString(fattVedtak.vedtakOppgaveDTO.referanse)
             )
             if (behandling == null) {
                 call.respond(HttpStatusCode.NotFound, "Fant ingen behandling")
             } else {
                 inTransaction {
-                    behandlingsstatusService.settFattetVedtak(behandling, attesterVedtakOppgave.vedtakHendelse)
-                    oppgaveService.haandterFattetvedtak(attesterVedtakOppgave.attesteringsOppgave)
+                    behandlingsstatusService.settFattetVedtak(behandling, fattVedtak.vedtakHendelse)
+                    oppgaveService.lukkOppgaveUnderbehandlingOgLagNyMedType(
+                        fattVedtak.vedtakOppgaveDTO,
+                        OppgaveType.ATTESTERING
+                    )
```

Comment:
Kanskje wrappe disse i feature-toggle for ny oppgaveliste?

Reply:
tar i egen PR